lib/index.js

```diff
@@ -26,16 +26,19 @@ class RtspClient extends EventEmitter {
     this.originalPacketLength = -1;
     this.packetLength = -1;
     this.packets = [];
+    this.packet_channel_id = -1;
```

Comment:
For consistency with codebase, I wouldn't mind having this be named packetChannelId, or just packetChannel.